graspologic/types.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+"""
+This module includes common graspologic type hint declarations.
```

Comment:
Is this also the file we'll do the grody-but-essential `if < 3.9 List = typing.List else list`?

Reply:
Yes, if required that's the best place for it. I think we should hold off until we officially add 3.9 to the supported versions list.

Reply:
FWIW is there anything else holding us back from adding 3.9 officially? Just curious, can't recall what else was broken or if we just haven't checked